### PR TITLE
LFS: minor fixes 2

### PR DESCRIPTION
**Pull request by no92**
Part of the mlibc LFS Project:tm:

### CODE REVIEW

File: options/posix/generic/arpa-inet-stubs.cpp [hunk 1]
```diff
@@ -107,19 +107,70 @@ int inet_aton(const char *string, struct in_addr *dest) {
...
+				}
+			}
+
+			size_t off = 0;
+			for(size_t i = 0; i < 8; i++) {
+				auto ptr = source->s6_addr + (i * 2);
+
+				/* if we are at the beginning of the largest block of zeroed quartets, place "::" */
+				if(i == max_zeroes_off && max_zeroes_len >= 2) {
+					if(off < size) dst[off++] = ':';
```
Comment (Dennisbonke) [inline options/posix/generic/arpa-inet-stubs.cpp:155]: Style issue, this should be
```c
if(off < size) {
	dst[off++] = ':';
}
```

File: options/posix/generic/arpa-inet-stubs.cpp [hunk 2]
```diff
@@ -107,19 +107,70 @@ int inet_aton(const char *string, struct in_addr *dest) {
...
+					if(off < size) dst[off++] = ':';
```
Comment (Dennisbonke) [inline options/posix/generic/arpa-inet-stubs.cpp:156]: Same as above.

File: options/posix/generic/arpa-inet-stubs.cpp [hunk 3]
```diff
@@ -107,19 +107,70 @@ int inet_aton(const char *string, struct in_addr *dest) {
...
+					i += max_zeroes_len - 1;
+
+					continue;
+				}
+
+				/* place a colon if we're not at the beginning of the string and it is not already there */
+				if(off && dst[off - 1] != ':') {
+					if(off < size) dst[off++] = ':';
```
Comment (Dennisbonke) [inline options/posix/generic/arpa-inet-stubs.cpp:164]: Same as above.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
3 style issues, other than that, LGTM.

**Review by Andy-Python-Programmer — APPROVED** [review]
Andy approved :)

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**